<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectionViewWidget</class>
 <widget class="QWidget" name="CollectionViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGraphicsView" name="placeholder"/>
   </item>
   <item row="1" column="0">
    <widget class="QGraphicsView" name="graphicsView_1"/>
   </item>
   <item row="1" column="1" rowspan="2" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_3"/>
   </item>
   <item row="1" column="3">
    <widget class="QGraphicsView" name="graphicsView_4"/>
   </item>
   <item row="2" column="0">
    <widget class="QGraphicsView" name="graphicsView_2"/>
   </item>
   <item row="2" column="3">
    <widget class="QGraphicsView" name="graphicsView_5"/>
   </item>
   <item row="3" column="0" rowspan="2" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_6"/>
   </item>
   <item row="3" column="2">
    <widget class="QGraphicsView" name="graphicsView_7"/>
   </item>
   <item row="3" column="3">
    <widget class="QGraphicsView" name="graphicsView_8"/>
   </item>
   <item row="4" column="2">
    <widget class="QGraphicsView" name="graphicsView_9"/>
   </item>
   <item row="4" column="3">
    <widget class="QGraphicsView" name="graphicsView_10"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
